integration-test/wait_for_postgres.py

```diff
@@ -0,0 +1,33 @@
+import os
+import time
+from contextlib import closing
+
+import psycopg2
+from psycopg2.sql import SQL, Identifier
+
+template = os.environ.get("CDC_POSTGRES_DSN_TEMPLATE", "postgres:///{database}")
+
+for i in range(1, 11):
```

Comment:
Can we not bake this retry logic into the test script instead?

Reply:
I am actually planning to bake this into the main cdc system. So it would do a few attempt before giving up.
But I would rather not mixing a change in the application logic in the same PR as a change builds an image that should replace the current production one. So when I swap the image in freight I can compare them to be the same.

Does it make sense to you?